includes/base/controls-stack.php

```diff
@@ -408,12 +408,14 @@ public function add_control( $id, array $args, $options = [] ) {
 
 		unset( $options['position'] );
 
-		if ( $this->current_popover && ! $this->current_popover['initialized'] ) {
-			$args['popover'] = [
-				'start' => true,
-			];
+		if ( $this->current_popover ) {
+			$args['popover'] = [];
+
+			if ( ! $this->current_popover['initialized'] ) {
+				$args['popover']['start'] = true;
 
-			$this->current_popover['initialized'] = true;
+				$this->current_popover['initialized'] = true;
+			}
```

Comment:
@matipojo 

It increases the widgets cache response from 2.84MB (241.62KB transferred) to 2.86MB (242.73KB transferred).